backend/lib/src/api/handlers/auth.rs

```diff
@@ -0,0 +1,405 @@
+use axum::{extract::State, http::StatusCode, response::IntoResponse, Json};
+use axum_jwt::Claims;
+
+use crate::{
+    error::Error,
+    models::auth::{JwtClaims, NonceRequest, VerifyRequest},
+    services::Services,
+};
+
+pub async fn nonce(
```

Comment:
Can you document all these functions to inform what are the expected inputs/outputs?

Reply:
Actually now that I write some, it's pretty verbose and not that useful. It's the implementation of the endpoints

Reply:
Add docs of when this endpoint is used